projects/canopy/src/lib/repsonsive/hide-at/hide-at.notes.ts

```diff
@@ -0,0 +1,54 @@
+export const notes = `
```

Comment:
Maybe we should write something in the grid notes to say these directives exist and add a link to these notes?

Reply:
Good idea 👍

Reply:
Added some notes here: https://github.com/Legal-and-General/canopy/pull/305/files#diff-9d96f01f73d60d459b090218b5baf1d53eb217acc3a66a143ca28476dad8ab71